Encounter Me.Api/Migrations/20211120134806_initial.cs

```diff
@@ -29,8 +29,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 name: "Users",
                 columns: table => new
                 {
-                    Id = table.Column<int>(type: "int", nullable: false)
-                        .Annotation("SqlServer:Identity", "1, 1"),
```

Comment:
What was that annotation `SqlServer:Identity` for? Is it not needed anymore because type of column now is `uniqueidentifier`?

Reply:
This file was auto-generated by EF core, I will check why it was still generated like that, even after reworking Db and fix it in the future.